app/controllers/concerns/accountable.rb

```diff
@@ -4,10 +4,11 @@ module Accountable
   extend ActiveSupport::Concern
   include SentryLogging
 
+  MHV_MAPPED_CSID = 'myhealthevet'
   def update_account_login_stats(login_type)
```

Comment:
Just a style thing but could you put a newline after the constant?
```suggestion
  MHV_MAPPED_CSID = 'myhealthevet'
  
  def update_account_login_stats(login_type)
```

Reply:
We should have a space after the constants, can you add the newline after this constant?